dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -0,0 +1,53 @@
+<LinkageCheckerFilter>
+  <!-- reactor-core's Traces catches Throwable to detect classes available in
+    Java 9+. As Linkage Checker targets Java 8, it ignores these errors to avoid
+    false positives.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/816 -->
+  <LinkageError>
+    <Source>
+      <Package name="reactor.core.publisher.Traces" />
+    </Source>
+  </LinkageError>
+
+  <!-- GraalVM-related libraries depend on Java Compiler Interface (JVMCI) that
+    only exists in special JDK. As Linkage Checker does not use GraalVM, it
+    ignores the errors to avoid false positives.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/929 -->
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="com.oracle.svm" />
+    </Source>
+  </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="com.oracle.graal" />
+    </Source>
+  </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="org.graalvm" />
+    </Source>
+  </LinkageError>
+
+  <!-- Mockito's MockMethodDispatcher uses special class loader to load
+    MockMethodDispatcher.raw. As Linkage Checker does not use the class loader,
+    it ignores the missing class errors to avoid false positives.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/407 -->
```

Comment:
There's no reason not to summarize the relevant parts of #407 here. I think there's more content there than in this file.

Reply:
Added the raw file is in the mockito JAR.